akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala

```diff
@@ -165,6 +165,11 @@ abstract class ActorSystem[-T] extends ActorRef[T] with Extensions with ClassicA
   def eventStream: ActorRef[EventStream.Command] =
     EventStreamExtension(this).ref
 
+  /**
+   * Obtain the external address of the default transport.
```

Comment:
I think some more details would be good, something like: "For a non clustered ActorSystem this will return an adress without host and port. For a clustered ActorSystem this will return the address that other nodes use to communicate with this node"